specs.md

```diff
@@ -167,16 +167,16 @@ BMS-6 | General voltage and temperature limits | The BMS must throw an AIR shutd
 BMS-7 | Charge temperature limits | The BMS must throw an AIR shutdown fault and enter the fault state if charging is attempted outside of these bounds: <br/>  0.0C < any cell temperature < 45.0C. | EV.5.1.3, EV.5.1.10
 BMS-9 | Charger detection and logging | - The BMS must check the charger connection status at 1Hz by the state of the CHARGE_STATE_3V3 digital input. <br/> - The BMS must log the charger connection status over CAN at 1Hz or faster.
 BMS-10 | Charger enable/disable | The BMS must enable the charger by setting the BMS PON pin high and disable the charger by setting the BMS PON pin low.
-BMS-11 | AIRs weld/stuck open detection | The BMS must check that the AIR+ and AIR- are in the desired open or closed state at 1kHz, and if not the BMS must throw an AIR shutdown fault and enter the fault state.
+BMS-11 | AIRs weld/stuck open detection | The BMS must check that the AIR+ and AIR- are in the desired open or closed state at 100Hz, and if not the BMS must throw an AIR shutdown fault and enter the fault state.
```

Comment:
i don't think we can detect whether AIR- is open or closed right?

Reply:
Oh whoops

Reply:
Wait I thought from this LUT that we could detect it, but cannot open/close.

![image](https://user-images.githubusercontent.com/32230980/95791062-6e974580-0c95-11eb-8261-6d67d94f003c.png)

Reply:
i did come up with that truth table... i have forgotten how this car works

Reply:
this is fine then